abis/linux/in.h

```diff
@@ -39,11 +39,34 @@ struct sockaddr_in6 {
 	uint32_t        sin6_scope_id;
 };
 
+struct ip_mreq {
+	struct in_addr imr_multiaddr;
+	struct in_addr imr_interface;
+};
+
 struct ipv6_mreq {
 	struct in6_addr ipv6mr_multiaddr;
 	unsigned        ipv6mr_interface;
 };
 
+struct in_pktinfo {
+	unsigned int ipi_ifindex;
+	struct in_addr ipi_spec_dst;
+	struct in_addr ipi_addr;
+};
+
+struct in6_pktinfo {
+	struct in6_addr ipi6_addr;
+	uint32_t ipi6_ifindex;
+};
+
+struct group_req {
+	uint32_t gr_interface;
+	struct sockaddr_storage gr_group;
+};
+
+extern const struct in6_addr in6addr_any;
```

Comment:
We already define this in `options/posix/include/netinet/in.h`, so I think this should be removed.